cppForSwig/BtcUtils.h

```diff
@@ -766,11 +775,44 @@ class BtcUtils
       return (8 + viLen + scrLen);
    }
 
+   static uint32_t TxWitnessCalcLength(uint8_t const * ptr)
+   {
+      uint32_t witLen = 0;
+      uint32_t viStackLen;
+      uint32_t stackLen = (uint32_t)readVarInt(ptr, &viStackLen);
+      witLen += viStackLen;
+      for(uint32_t i=0; i<stackLen; i++)
+      {
+         uint32_t viLen;
+         witLen += (uint32_t)readVarInt(ptr+witLen, &viLen);
+         witLen += viLen;
+      }
+      return witLen;
+   }
+
+   static size_t TxWitnessCalcLength(uint8_t const * ptr, size_t size)
+   {
+       if (size < 1)
+        throw BlockDeserializingException();
+       uint32_t witLen = 0;
+       uint32_t viStackLen;
+       uint32_t stackLen = (uint32_t)readVarInt(ptr, size, &viStackLen);
+       witLen += viStackLen;
+       for(uint32_t i=0; i<stackLen; i++)
+       {
+          uint32_t viLen;
+          witLen += (uint32_t)readVarInt(ptr+witLen, size-witLen, &viLen);
+          witLen += viLen;
```

Comment:
same here, should check there is enough room left in the buffer to cover for the viLen